pkg/handlers/admin_entities.go

```diff
@@ -33,8 +33,8 @@ var startTime = time.Now()
 // JSON is a map alias, just for convenience
 type JSON map[string]interface{}
 
-// Health is holding info about health checks
-type Health struct {
+// HealthBody is holding info about health checks
+type HealthBody struct {
```

Comment:
Why changed name?

Reply:
since we have declared a new handler called `Health` here, we cannot have a struct with the same name: https://github.com/optimizely/agent/pull/232/files#diff-dd19b841dc70477024a581f7481b9784R27

Reply:
I'd prefer something like `HealthData` or `HealthInfo` over `HealthBody`.

Reply:
Let's move this struct into the new `Health` handler and remove this `admin.Health` handler. That would mean modifying `routers/admin.go` to use this new `Health` handler.